packages/status-page/src/utils/getNumProvers.ts

```diff
@@ -11,8 +11,10 @@ export type UniqueProverResponse = {
 export const getNumProvers = async (
   eventIndexerApiUrl: string
 ): Promise<UniqueProverResponse> => {
-  const resp = await axios.get<UniqueProverResponse>(
+  const uniqueProverResp = await axios.get<UniqueProverResponse>(
     `${eventIndexerApiUrl}/uniqueProvers`
   );
-  return resp.data;
+  uniqueProverResp.data.provers.sort((a, b) => b.count - a.count);
```

Comment:
Lets also hide the prover with `address(0)` since its from an event of our oracle prover (who does not submit the actual zk proof)?